scene/mission-replay/src/main/java/com/esri/samples/scene/AnimationModel.java

```diff
@@ -0,0 +1,90 @@
+package com.esri.samples.scene;
+
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+
+/**
+ * Model bean to bind to animation properties.
+ */
+public class AnimationModel {
+
+  private final IntegerProperty frames;
+  private final IntegerProperty keyframe;
+
+  /**
+   * Default constructor (needed for FXML injection).
+   */
+  public AnimationModel() {
+    this.frames = new SimpleIntegerProperty(1);
+    this.keyframe = new SimpleIntegerProperty(0);
+  }
+
+  /**
+   * Constructs the animation model with the specified keyframe.
+   *
+   * @param keyframe starting animation frame
+   */
+  public AnimationModel(int keyframe) {
+    this();
+    this.setKeyframe(keyframe);
+  }
+
+  public int getFrames() {
```

Comment:
Some javadoc would be good.
Should it be getFrame() ? maybe getCurrentFrame() ?